src/Yoti/YotiClient.php

```diff
@@ -430,20 +359,20 @@ private function decryptConnectToken($encryptedConnectToken)
      *
      * @param string bool|$pem
      *
-     * @throws \Exception
+     * @throws YotiClientException
      */
     private function processPem(&$pem)
     {
         // Check PEM passed
         if(!$pem)
         {
-            throw new \Exception('PEM file is required', 400);
+            throw new YotiClientException('PEM file is required', 400);
```

Comment:
It looks like every time we throw an Exception, we give it a http status code.  Is this a PHP requirement?  What are they for?

Reply:
This is for client to handle the request based on the HTTP code, imo each HTTP code has a meaning. So me people will be referring to the HTTP code not even the actual message.
If you think the HTTP code is not the right one, that's fine we can change, but don't tell me it's unnecessary.